install/03_rbac.yaml

```diff
@@ -221,3 +221,37 @@ subjects:
   - kind: ServiceAccount
     name: cluster-autoscaler
     namespace: openshift-machine-api
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: prometheus-k8s
```

Comment:
A bit concerned about naming of the Role and RoleBinding. We share a namespace with all the other cluster-api related components. Will they each create this when they have monitoring enabled? If so, they should probably have the operator name included here to create separate resources, even if the permissions are the same.

Reply:
great point